gateway/src/node/client_handling/active_clients.rs

```diff
@@ -147,6 +154,9 @@ impl ActiveClientsStore {
         if self.inner.insert(client, entry).is_some() {
             panic!("inserted a duplicate remote client")
         }
+        let _ = self
+            .stats_event_sender
+            .unbounded_send(events::new_session_start_event(client));
```

Comment:
Same as comment above

Reply:
Fixed in adf198f6b8696b963643253dd0ee3aa6e745f13c